documentation/milestones/individual_milestones/static_site.md

```diff
@@ -0,0 +1,199 @@
+  - [ ] The site adopts the UI principles and framework established in the [Foundational UI milestone](https://github.com/HHS/grants-equity/issues/60)
+  - [ ] The site is 508 compliant and satisfies the other guidelines outlined in the Accessibility Planning milestone
+- [ ] The following content requirements are satisfied:
+  - [ ] The site contains information about both the NOFO Transformation and Grants.gov Modernization workstreams
+  - [ ] The content on the site has been been reviewed and approved by the relevant stakeholders within each workstream
+  - [ ] The site also links to external resources related to the project (if they are available), including:
+    - [ ] The legacy grants.gov site
+    - [ ] The open source repository
+    - [ ] The open source wiki
+    - [ ] The open source chat
+  - [ ] Stretch goal: The site provides a mechanism to collect stakeholder feedback or questions
+
+### Proposed metrics for measuring goals/value/definition of done
+
+1. Number of unique site visitors
+2. Total number of site visits
+3. Stretch: Number of form responses
+4. Uptime service
+5. [Lighthouse score](lighthouse) for the site
+6. Deployment build time
+7. Deployment/hosting costs
+
+### Destination for live updating metrics
+
+Page on the public wiki. **Note:** This will likely change once we deliver [the Public Measurement Dashboard milestone](../milestone_short_descriptions.md#public-measurement-dashboards)
+
+## Planning
+
+### Assumptions & dependencies
+<!-- Required -->
+
+*What capabilities / milestones do we expect to be in place at the beginning of work on this milestone?*
+
+- [ ] **Front-end Planning:** Determines the language, framework, and deployment service used to build and host the site.
+- [ ] **Developer Tools:** Establishes a suite of tools used to ensure the quality and security of the site codebase.
+- [ ] **beta.grants.gov Domain:** Secures access to the `beta.grants.gov` domain which is where the site will be hosted.
+- [ ] **Authority to Operate (ATO):** Ensures that the site and the infrastructure that hosts it are comply with the Software Security Plan for the legacy grants.gov site.
+- [ ] **Infrastructure-as-Code:** Programmatically provisions the resources needed to deploy and host this site.
+- [ ] **Front-end CI/CD:** Sets up a CI/CD pipeline that will be used to test and publish code changes to the site.
+- [ ] **Foundational UI:** Determines the UI framework that the site will adopt before launch.
+- [ ] **Web Analytics:** Enables tracking key success metrics for this milestone, e.g. site traffic and number of unique visitors.
+
+*Are there any notable capabilities / milestones do NOT we expect to be in place at the beginning of work on this milestone?*
+
+- [x] **Internationalization:** While there will be content delivered within this milestone that needs to be translated in the future, we do not expect to have a framework for managing translations set up by the time this milestone is delivered.
```

Comment:
I'm fine kicking the can on translations, but do we know when we might want to put translated content in PROD? It would be great to know roughly when that's on the calendar.